spec.md

```diff
@@ -99,7 +97,8 @@ The `Span` interface must have the following capabilities:
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
+- Start a new child `Span` with a given operation name. With no parent, commonly referred to as a *root* `Span`, no trace attributes exist. With a parent any trace attributes must be passed through into the created child. **(py: `create_span`, go: `CreateSpan`)**
```

Comment:
@dkuebric 

When in a tomcat process would you be creating spans off one sampling logic at the controller level but creating spans (with no parent span) off a different sampling logic?

I can't get my head around that being a valid use-case, and in the worse and unlikely situation one could just construct a separate tracer in the jdbc layer.